src/makeInfinityAdder.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
-}
+  let sum = 0;
+  let count = 0;
```

Comment:
Why do you need this count?

Reply:
It was created to check if there was an empty function call after an empty function call 
<img width="457" alt="image" src="https://user-images.githubusercontent.com/104283893/193454264-edfa4f31-6986-470c-9226-c2464b16873b.png">

Reply:
if there were previous calls and the function was return + the current call is not a number (in other words, empty call) then we will return 0 to reset